src/components/mdx/ProseSection.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const ProseSection = ({children}) => {
```

Comment:
**thought:** allowing additional classes to be added would be pretty handy for this one I think.